### PR TITLE
Allow to use remote ssl certs

### PR DESCRIPTION
**Pull request by Fobhep**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Fobhep** [comment]
@domenicbove as discussed

File: hosts_example.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ all:
     ## As well as the signed certificate path and the key for that certificate for each host.
     ## These will need to be set for the correct host
     # ssl_custom_certs: true
+    # ssl_remote_certs: true # if you set this to true, ansible will resolve the paths on each kafka-broker to find key, crt and ca file
```
Comment (domenicbove) [inline hosts_example.yml:46]: Can you change this var to `ssl_custom_certs_remote_src`
And change the description to `set to true if key crt and ca file already on hosts, file paths still must be set'

File: roles/confluent.ssl/tasks/custom_certs.yml [hunk 2]
```diff
@@ -1,16 +1,19 @@
 ---
 - name: Copy CA Cert to Host
   copy:
+    remote_src: "{{ssl_remote_certs | default(omit)}}"
```
Comment (domenicbove) [inline roles/confluent.ssl/tasks/custom_certs.yml:4]: could also just default `ssl_custom_certs_remote_src: false` in a variables file

**Comment by domenicbove** [comment]
@Fobhep I created this pr with the changes I requested: https://github.com/ATIX-AG/cp-ansible/pull/2
And tested it does not break the general custom certs flow.

Please review and if we merge that one, then we can merge this one!

**Pull request merged**